controllers/contact-controller.js

```diff
@@ -8,6 +8,12 @@ const contactModel = require('../models/contact.js');
 
 const { Op } = Sequelize;
 
+// Intentionally adding only the types of interest here
+const callTypes = {
+  child: 'Child calling about self',
+  caller: 'Someone calling about a child',
+};
+
```

Comment:
This is a (partial) copy paste of the callTypes from flex-plugin. Should we define a constants file in this repository? Or how are we going to handle this "shared" constants?